internal/cli/users_roles.go

```diff
@@ -0,0 +1,65 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/spf13/cobra"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+)
+
+func userRolesCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage a user's roles",
+		Long: "Manage a user's assigned roles. To learn more about roles and their behavior, read " +
+			"[Role-based Access Control](https://auth0.com/docs/manage-users/access-control/rbac).",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(showUserRolesCmd(cli))
+
+	return cmd
+}
+
+func showUserRolesCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "show",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Show a user's roles",
+		Long:  "Display information about an existing user's assigned roles.",
+		Example: `  auth0 users roles show
+  auth0 users roles show <user-id>
+  auth0 users roles show <user-id> --json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			var userRoleList *management.RoleList
+			if err := ansi.Waiting(func() (err error) {
+				userRoleList, err = cli.api.User.Roles(inputs.ID)
+				return err
+			}); err != nil {
+				return fmt.Errorf("failed to find roles for user with ID %s: %w", inputs.ID, err)
+			}
+
+			cli.renderer.RoleList(userRoleList.Roles)
```

Comment:
Currently this none-found hint is displayed: "Use 'auth0 roles create' to add one"

Eventually this help text should display something regarding the assignment of a role. Ex: "Use `auth0 users roles assign`"

Reply:
Also: "No roles available." should read something like "No roles assigned to user".

Reply:
This now displays in [`3094faf` (#604)](https://github.com/auth0/auth0-cli/pull/604/commits/3094faf55e643bffabf1603e25a97906febd7593)

```
=== tenant.eu.auth0.com roles assigned to user (0)

No roles assigned to user available.

 ▸    Use 'auth0 users roles assign' to add one
```